src/mushroom-strategy.ts

```diff
@@ -175,9 +175,17 @@ class MushroomStrategy extends HTMLTemplateElement {
                 deviceOptions = Helper.strategyOptions.card_options?.[entity.device_id];
               }
 
-              if (!cardOptions?.hidden && !deviceOptions?.hidden) {
-                domainCards.push(new cardModule[className](entity, cardOptions).getCard());
+              if (cardOptions?.hidden || deviceOptions?.hidden) {
+                continue;
               }
+
+              if (entity.entity_category === "config" &&
+                (Helper.strategyOptions.domains[domain].hide_config_entities ||
+                  Helper.strategyOptions.domains.default.hide_config_entities)) {
```

Comment:
Whats the purpose of `|| Helper.strategyOptions.domains.default.hide_config_entities` ?

Reply:
It allows a blanket hide option for all domains.

Reply:
I see, however domain `default` represents the _Miscellaneous_ domain internally.

We've used `_` as an identifier at the area options to reference all areas.
You can do the same for referencing all domains.